client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -8,22 +8,74 @@ import { versionType } from 'types/versionType';
 
 class HistoryViewerVersionDetail extends PureComponent {
   componentWillMount() {
-    this.toggleToolbarClass();
+    this.toggleToolbarClass(true);
   }
 
   componentWillUnmount() {
-    this.toggleToolbarClass();
+    this.toggleToolbarClass(false);
+  }
+
+  /**
+   * A useful doc block explaining why we construct an array of versions in each case (noting that
+   * it previously was hard coded to return an array containing the current version only).
+   *
+   * @returns {Array}
+   */
+  getListVersions() {
+    const { compare, version } = this.props;
+    if (this.isCompareMode()) {
+      return [compare.versionTo, compare.versionFrom];
+    }
+    return [version];
+  }
+
+  /*
+   * Return whether or not we should be displaying the preview component
+   * @return bool
+   */
+  isPreviewable() {
+    const { isPreviewable } = this.props;
+    return isPreviewable && !this.isCompareMode();
+  }
+
+  /*
+   * Return whether or not we should be comparing two versions
+   * @return bool
```

Comment:
It's `@returns` in JSDoc =( `@returns {boolean}`

Reply:
thanks :)